src/components/flexjar/Flexjar.tsx

```diff
@@ -51,6 +51,19 @@ export const Flexjar = ({ side }: FlexjarProps) => {
     }
   }, [emojiType]);
 
+  const openFeedbackPanel = () => {
+    setApen(!apen);
```

Comment:
Når man trykker på åpne-knappen, setter vi status til å ikke være åpen? 🤔

Reply:
Hmm ja det er vel navngivningen som blir rar her, burde kanskje vært `isApen` og `setIsApen` siden man bare toggler den. Og kanskje `toggleFeedbackPanel` i stedet for `openFeedbackPanel` 👍🏼